.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         os:
           - ubuntu-latest
         toolchain:
-          - nightly
+          - stable
     steps:
```

Comment:
something strange is going on with the Github checks?

<img width="865" alt="Screenshot 2023-11-08 at 12 10 17" src="https://github.com/tablelandnetwork/basin-provider/assets/5305984/21c37fa3-a1b4-4ac4-bf2a-f606fb29cb3f">